mh/constants.py

```diff
@@ -257,6 +255,7 @@ def slot(item, qty):
 OTHER_CAPCOM_ID = b"D9R7K4"
 OTHER_HUNTER_NAME = b"Drakea"
 
+
```

Comment:
Is this newline really needed? I don't recall PEP8 to blame this line.

Reply:
Removed it